docs/source/tracking.rst

```diff
@@ -433,3 +433,39 @@ then make API requests to your remote tracking server.
     with mlflow.start_run():
         mlflow.log_param("a", 1)
         mlflow.log_metric("b", 2)
+
+.. _system_tags:
+
+System tags
+===========
+Runs can be annotated with arbitrary tags. Tag keys that start with ``mlflow.`` are reserved for
+internal use. The following tags are set automatically by MLflow, when appropriate:
+
++-------------------------------+----------------------------------------------------------------------------------------+
+| Key                           | Description                                                                            |
++===============================+========================================================================================+
+| ``mlflow.runName``            | Human readable name that identifies this run.                                          |
```

Comment:
How about giving context on how the system set it. Like-

Human readable name that identifies this run. Automatically set when provided as an argument to `mlflow.start_run`.